Task6/src/main/java/arrays/ArrayListAndArray.java

```diff
@@ -0,0 +1,60 @@
+package arrays;
+
+import java.util.ArrayList;
+import java.util.Date;
+
+public class ArrayListAndArray {
+    private String[] array =new String[10];
+    private ArrayList<String> list = new ArrayList<String>();
+    long timeArrayAdd100k;
+    long timeListAdd100k;
+
+    public void addStrings(String[] array, String newString){
+        int arrayLength = array.length;
+        int newSize = arrayLength * 2;
+
+        if (arrayLength == newSize){
+            String[] newArray = new String[newSize];
+            System.arraycopy(array, 0, newArray, 0, arrayLength);
+            array[arrayLength] = newString;
+}
+    }
+
+    public void add100kStringsToArray(){
+        Date startArray = new Date();
```

Comment:
Rename variable to startArrayTime please. Do the same for all similar variables.

Reply:
done